tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -106,66 +113,86 @@
+                        </ic:EventTriggerBehavior>
+                    </i:Interaction.Behaviors>
+                </AutoSuggestBox>
+                <StackPanel Grid.Column="2" Orientation="Horizontal" HorizontalAlignment="Right">
+                    <!-- Provider field -->
+                    <TextBlock x:Uid="ProviderTextBlock" VerticalAlignment="Center" Margin="50 0 5 0"/>
+                    <ComboBox x:Uid="ProviderSelectionComboBox" x:Name="ProviderSelectionComboBox" Margin="0 3 5 0"
+                          ItemsSource="{x:Bind ViewModel.Providers}" SelectedIndex="{x:Bind ViewModel.SelectedProviderIndex, Mode=TwoWay}">
+                        <i:Interaction.Behaviors>
+                            <ic:EventTriggerBehavior EventName="SelectionChanged">
+                                <ic:InvokeCommandAction Command="{x:Bind ViewModel.ProviderHandlerCommand}" />
+                            </ic:EventTriggerBehavior>
+                        </i:Interaction.Behaviors>
+                    </ComboBox>
+                    <!-- Sort field -->
+                    <TextBlock x:Uid="SortByTextBlock" VerticalAlignment="Center" Margin="10 0 5 0"/>
+                    <ComboBox x:Uid="SortSelectionComboBox" x:Name="SortSelectionComboBox" Margin="0 3 13 0"
+                          SelectedIndex="{x:Bind ViewModel.SelectedSortIndex, Mode=TwoWay}">
+                        <ComboBoxItem x:Uid="SortByName"/>
+                        <ComboBoxItem x:Uid="SortByNameDesc"/>
+                        <ComboBoxItem x:Uid="SortByAltName"/>
+                        <ComboBoxItem x:Uid="SortByAltNameDesc"/>
+                        <ComboBoxItem x:Uid="SortByLastConnected"/>
+                        <i:Interaction.Behaviors>
+                            <ic:EventTriggerBehavior EventName="SelectionChanged">
+                                <ic:InvokeCommandAction Command="{x:Bind ViewModel.SortHandlerCommand}"/>
+                            </ic:EventTriggerBehavior>
+                        </i:Interaction.Behaviors>
+                    </ComboBox>
+                    <!--  'Sync' button -->
+                    <Button Padding="40 5 30 8" Command="{x:Bind ViewModel.SyncButtonCommand}" Margin="0 3 13 0">
+                        <StackPanel Orientation="Horizontal">
+                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE72C;"/>
+                            <TextBlock x:Uid="SyncButtonTextBlock" Margin="8 0 0 0"/>
+                        </StackPanel>
+                    </Button>
+                </StackPanel>
+        </Grid>
+
+        <!-- Last synced text block-->
+        <StackPanel Grid.Row="2" Orientation="Horizontal" HorizontalAlignment="Right" Margin="0 12 15 0" >
+            <TextBlock x:Uid="LastSyncTextBlock"
+                       VerticalAlignment="Center"
+                       Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                       Margin="0 0 3 0"/>

```

Comment:
Then if you want the margin above and below you can put it back here: 
```suggestion
                       Margin="0 12 3 12" />
```
(Or do something different, but this gets you something that looks nice quickly)